src/panels/config/blueprint/ha-blueprint-overview.ts

```diff
@@ -326,7 +327,19 @@ class HaBlueprintOverview extends LitElement {
   };
 
   private _delete = async (ev) => {
-    const blueprint = ev.currentTarget.blueprint;
+    const blueprint = ev.currentTarget.blueprint as BlueprintMetaDataPath;
+    const related = await findRelated(
+      this.hass,
+      `${blueprint.domain}_blueprint`,
+      blueprint.path
+    );
+    if (related.automation?.length || related.script?.length) {
+      showAlertDialog(this, {
```

Comment:
Should we ask the user if we should delete all scripts/automations that use the blueprint for them? Seems a bit scary...

Reply:
Convert? What do you mean with that?